policyengine_us/variables/gov/states/in/tax/income/credits/in_refundable_credits.py

```diff
@@ -0,0 +1,15 @@
+from policyengine_us.model_api import *
+
+
+class in_refundable_credits(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Indiana refundable income tax credits"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://iga.in.gov/laws/2021/ic/titles/6#6-3.1"
+    defined_for = StateCode.IN
+
```

Comment:
Replacing `formula`, unless the `in` causes this to fail?
```suggestion
    adds = "gov.states.in.tax.income.credits.refundable"
```

Reply:
As I remember, it did fail, hence the use of the formula.
To do what you are suggesting, you would have to get @nikhilwoodruff to enhance PolicyEngine-Core.

Reply:
Did you try `adds = "gov.states['in'].tax.income.credits.refundable"`?

Reply:
Also I filed https://github.com/PolicyEngine/policyengine-core/issues/112 for a core workaround